.editorconfig

```diff
@@ -0,0 +1,15 @@
+# See: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/formatting-rules
+
+[*.cs]
+indent_style=tab
+indent_size=tab
```

Comment:
Aaaaargh! Nooooooo! Spaces, please.

Reply:
https://www.youtube.com/watch?v=SsoOG6ZeyUI&t=80s

Reply:
Indeed one of the better parts of SV...